lab_5_scrapper/scrapper.py

```diff
@@ -2,10 +2,46 @@
 Crawler implementation.
```

Comment:
please, check https://github.com/fipl-hse/2023-2-level-ctlr/actions/runs/8720761911/job/23923160933?pr=25

you need to fix your import style

Reply:
Fixed